src/index.js

```diff
@@ -10,13 +10,29 @@ nunjucks.configure("views", {
 
 app.use(express.static("public"));
 
-app.get("/", function(request, response) {
+app.get("/", async function(req, res) {
+  console.log(req.query.page);
```

Comment:
I would replace this with something more appropriate such as "someone requesting page /" maybe even add the page they are requesting